testing/test_capture.py

```diff
@@ -1510,3 +1510,27 @@ def test_fails():
     result_with_capture.stdout.fnmatch_lines(
         ["E           TypeError: write() argument must be str, not bytes"]
     )
+
+def test_capsysord(capsysord):
+    """Show that capsysord is capable of preserving chronological order of streams."""
+    print("stdout1")
+    print("stdout2")
+    print("stderr1", file=sys.stderr)
+    print("stdout3")
+    print("stderr2", file=sys.stderr)
+    print("stderr3", file=sys.stderr)
+    print("stdout4")
+    print("stdout5")
+
+    out, err = capsysord.readouterr(combined=True)
+    assert (
+        out == "stdout1\n"
+        "stdout2\n"
+        "stderr1\n"
+        "stdout3\n"
+        "stderr2\n"
+        "stderr3\n"
+        "stdout4\n"
+        "stdout5\n"
+    )
```

Comment:
Shoudl also test non-combined output (and make the assert more readable.. :))

Reply:
This test was more a demo than real test, I'll also cover the non-combined use case. What's wrong with the asserts?

Reply:
> What's wrong with the asserts?

It's actually ok.